### ISSUE TITLE
Add augmentations used by the image recognition methods

### ISSUE THREAD

**Issue by tarvaina** [AI-metrics/master_text#62]
In order to make comparing different image recognition methods easier, it would help if the tables and charts included the augmentations used by the papers. Image recogntion can be made easier by augmenting the input data with rotated, flipped, color-distorted, etc. versions of the images. Research papers often intentionally limit augmentation to keep the problems hard and to make comparisons between papers easier.

E.g. from the list it looks like CIFAR-10 is best solved by fractional max pooling. However there has been lots of progress since then; the paper just used a more extensive set of image augmentations than most of the other papers.

For CIFAR-10, the standard set of image augmentation applied image research papers appears to be random horizontal flips and image translations. It would be helpful to distinguish papers that use only these augmentations from the papers that use a more extensive set of augmentations. I did not check, but I assume this applies to other image recognition tasks and tables as well.

**Comment by pde** [comment]
Hi @tarvaina yes, this sounds like a good idea. The bAbi 20 qa metrics are probably an example where (thanks to input from researchers in that field, see #35) we've done a better job distinguishing variations in training and augmentation.

You'll see from that case that the Notebook now supports a couple of methods for helping distinguish things.  One is to add `not_directly_comparable=True` to the `.measure()` calls for results that use unusual forms of training or augmentation.  That will cause them to be rendered as a black star and not considered part of the frontier of best results.

Another, if there are several widely used variants of the metric or training data, is to split them into two `Metric` instances but put them on the same graph as we did with the bAbi1k and bAbi10k variants and the 1.0 and 2.0 versions of the VQA real dataset.

A third example of delineating variation in training setups is in the Atari data, where we label data points with "hs" or "noop" to indicate a comparatively unimportant variation in training, which doesn't  necessarily seem to make the metrics easier or harder in general.

We'd love to have pull requests that start using these methods for CIFAR-10 and other vision problems!
